src/test/java/org/ballerinalang/tests/connectors/http/ContentEncodingTests.java

```diff
@@ -0,0 +1,107 @@
+package org.ballerinalang.tests.connectors.http;
+
+import org.apache.commons.httpclient.methods.InputStreamRequestEntity;
+import org.ballerinalang.client.HttpTestClient;
+import org.ballerinalang.client.ResponseHolder;
+import org.ballerinalang.client.compressor.GzipCompressor;
+import org.ballerinalang.tests.base.BallerinaBaseTest;
+import org.testng.annotations.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+
+/**
+ * Test class for testing transfer and content encoding
+ * Uses EncodingService.bal
+ */
+public class ContentEncodingTests extends BallerinaBaseTest {
+
```

Comment:
Remove random whitespace, fix in all the places.

Reply:
This is fixed with #10